internal/providers/pluginfw/tfschema/float64_attribute.go

```diff
@@ -40,7 +40,7 @@ func (a Float64AttributeBuilder) BuildResourceAttribute() schema.Attribute {
 	}
 }
 
-func (a Float64AttributeBuilder) SetOptional() AttributeBuilder {
+func (a Float64AttributeBuilder) SetOptional() BaseSchemaBuilder {
```

Comment:
Does it work if these functions return `Float64AttributeBuilder`, or does the return type need to be the interface?

Reply:
That works, but I preferred this to comply with the interface of BaseSchemaBuilder